internal/daemon/controller/handlers/targets/target_service.go

```diff
@@ -1305,7 +1305,21 @@ func (s Service) createInRepo(ctx context.Context, item *pb.Target) (target.Targ
 	if err != nil {
 		return nil, nil, nil, err
 	}
-	out, err := repo.CreateTarget(ctx, u)
+
+	createOptions := []target.Option{}
```

Comment:
Do we need to do some kind of check to ensure the user is authorized to create aliases?